### PR TITLE
BMCP-5: About page

### PR DESCRIPTION
**Pull request by nikhilsharma0025**
## Description 
Added  About US Page 

#### Modification
Added about us component

### CODE REVIEW

**Comment by truelegion47** [comment]
@nikhilsharma0025 How much more time are we going to spend on this ?

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
...circle fill='%2300c7f1' cx='50' cy='300' r='50'/%3E%3Cg fill='%2300beee' %3E%3Ccircle cx='0' cy='350' r='50'/%3E%3Ccircle cx='100' cy='350' r='50'/%3E%3C/g%3E%3Ccircle fill='%2300b4e9' cx='50' cy='400' r='50'/%3E%3Cg fill='%2300abe5' %3E%3Ccircle cx='0' cy='450' r='50'/%3E%3Ccircle cx='100' cy='450' r='50'/%3E%3C/g%3E%3Ccircle fill='%2300a2df' cx='50' cy='500' r='50'/%3E%3Cg fill='%230099da' %3E%3Ccircle cx='0' cy='550' r='50'/%3E%3Ccircle cx='100' cy='550' r='50'/%3E%3C/g%3E%3Ccircle fill='%230090d3' cx='50' cy='600' r='50'/%3E%3Cg fill='%230087cc' %3E%3Ccircle cx='0' cy='650' r='50'/%3E%3Ccircle cx='100' cy='650' r='50'/%3E%3C/g%3E%3Ccircle fill='%23007ec5' cx='50' cy='700' r='50'/%3E%3Cg fill='%230d75bd' %3E%3Ccircle cx='0' cy='750' r='50'/%3E%3Ccircle cx='100' cy='750' r='50'/%3E%3C/g%3E%3Ccircle fill='%23216bb5' cx='50' cy='800' r='50'/%3E%3Cg fill='%232d62ac' %3E%3Ccircle cx='0' cy='850' r='50'/%3E%3Ccircle cx='100' cy='850' r='50'/%3E%3C/g%3E%3Ccircle fill='%233559a3' cx='50' cy='900' r='50'/%3E%3Cg fill='%233b5099' %3E%3Ccircle cx='0' cy='950' r='50'/%3E%3Ccircle cx='100' cy='950' r='50'/%3E%3C/g%3E%3Ccircle fill='%2340478f' cx='50' cy='1000' r='50'/%3E%3C/g%3E%3C/svg%3E");
```
Comment (2SkeePeR2) [inline src/App.css:4]: Why is it not styled?

File: src/App.tsx [hunk 2]
```diff
@@ -2,11 +2,24 @@ import { Topbar } from './components/Topbar/Topbar'
...
 
 export const App: React.FC = () => {
   return (
     <div className="App">
       <Topbar />
+      <Router>
+        <Switch>
+          <Route path='/' component={Home} />
+          <Route path='/about' component={About} />
+          {/* <Route path='/srvices' component={Services} />
```
Comment (2SkeePeR2) [inline src/App.tsx:17]: Comments ?

File: src/components/About/About.tsx [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+            <h3>
+                Incepted in Yr.2016. BMCP is a fast growing neutral service provider in Mobile Handset, Consumer Electronics,
+                Home Appliances, IT Solutions & hardware and Office Automation. We support our customers to do business better by leveraging our deep
+                customer service experience , experienced team , process driven approach , comprehensive portfolio of services and vertically aligned
+                business model. Multi brand service capability and strong service management process makes us a preferred destination for customers looking
+                for a long term solution to their service related needs We have grown YoY in C-sat , Customer call nos: managed and revenues since Yr.2016.
+                Multi-brand service capability and strong service management system (B-smarT)
+                make us a preferred destination for customers looking for a long term solution to their service-related needs
+            </h3>
+            <br></br>
```
Comment (2SkeePeR2) [inline src/components/About/About.tsx:42]: Avoid using ```<br>```. Instead use styles like padding or margin to space and place objects.

File: src/components/About/About.tsx [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+                    Based in the heart of India we aim high at being focused on building relationships with our clients and community. BMCP provides end to
+                    end customer care solutions for mobile and wearable devices. We help many of the world’s leading mobile and wearable device companies deliver
+                     a superior aftersales service experience. Using our creative gifts drives this foundation. Our experience allows clients to optimize the entire 
+                    customer care journey of consumers with their devices by ensuring an excellent and consistent customer care experience.
+            </h3>
+            </div>
+            <h1 className={classes.header} >We Value</h1>
+            <Grid container className={classes.root} spacing={2} color="primary">
+                <Grid item xs={2}>
+                            <Card className={classes.svg}>
```
Comment (2SkeePeR2) [inline src/components/About/About.tsx:53]: How can a className have ```.svg``` ? Please use it only as an extension and rename this.

File: src/components/About/About.tsx [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+                                    <Typography gutterBottom variant="h5" component="h2">
+                                      <b> Manager Operations</b>
+                                    </Typography>
+                                    <Link href="https://www.linkedin.com/in/rishab-sahani-864ab6100/"  ><LinkedInIcon className={classes.icon} /></Link>
+                                </CardContent>
+                            </Card>
+                        </Grid>
+                    </Grid>
+                            </Grid>
+                        </Grid>
```
Comment (2SkeePeR2) [inline src/components/About/About.tsx:207]: Fix the styling

File: src/components/Home/Home.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const Home:React.FC=()=>{
+    return (
+     <div>
+            <h1>  Home</h1>   
+         </div>
```
Comment (2SkeePeR2) [inline src/components/Home/Home.tsx:7]: Styling ?

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]
Pictures are not be uploaded to Github. Although for now it's OK but after the website is finished, consider moving it to an online picture library, creating a link to it, and using its URL.

All pictures must be gitignored.

File: src/App.tsx [hunk 7]
```diff
@@ -2,11 +2,24 @@ import { Topbar } from './components/Topbar/Topbar'
...
+                <Route path='/footer' component={Footer} />
+                <Route path='/footer' component={Footer} /> */}
+          <Route path='/footer' component={Footer} />
```
Comment (nik-1236789) [inline src/App.tsx:20]: Footer isn't a Route.

File: src/components/About/About.tsx [hunk 8]
```diff
@@ -0,0 +1,210 @@
+import { Avatar, Button, Card, CardActionArea, CardActions, CardContent, CardMedia, Grid, Link, Slide, SvgIcon, SvgIconProps, Typography } from '@material-ui/core';
+import React from 'react';
+import useStyles from './index'
+import manojWadwan from '../../images/manojWadwan.png'
```
Comment (nik-1236789) [inline src/components/About/About.tsx:4]: For now this is fine but when you are finished with the website, lets host these images online and get their urls.

**Review by nik-1236789 — CHANGES_REQUESTED** [review]
Make the changes as discussed and do add a screenshot (on priority) of the work that is done here.

File: src/components/About/About.tsx (hunk 3 above)
Comment (nikhilsharma0025) [reply src/components/About/About.tsx:42]: Ohk

**Pull request merged**